lib/system.nim

```diff
@@ -2810,6 +2810,7 @@ proc arrayWith*[T](y: T, size: static int): array[size, T] {.raises: [].} =
       result[i] = `=dup`(y)
 
 
-when defined(nimV2) and hasThreadSupport:
-  from system/arc import isUniqueRef
-  export isUniqueRef
+when defined(nimV2):
+  when notJSnotNims and hasThreadSupport and hasAlloc and arcLikeMem and not declared(isUniqueRef):
+    from system/arc import isUniqueRef
```

Comment:
We have `when defined(nimV2): include system/arc` above, so `isUniqueRef` is already exported, and `not declared(isUniqueRef)` is always false, so this block doesn't do anything.

Reply:
Yes I seen it, but isUniqueRef is not declared in system devel.

Reply:
Yes it is?

```
> .\koch temp --lib:lib --warning[BareExcept]:off r --eval:"echo isUniqueRef (ref Exception)()"
...
true
```